contrib/objcwrapper/src/objcwrapper.nit

```diff
@@ -55,7 +59,19 @@ for arg in opts.rest do
 
 	# Check for errors
 	if root isa NError then
-		print_error "Syntax Error: {root.message}: {root.position or else ""}"
+		var pos = root.position
+		print_error "Syntax Error: {root.message}, at {pos or else ""}"
+		print_error "in {src}"
+		if pos != null then
+			var lines = content.split("\n")
+			for line in [pos.line_start..pos.line_end] do
+				print_error lines[line-1]
+			end
+
+			var ptr = " "*(pos.col_start-1).max(0) + "^"*(pos.col_end-pos.col_start)
+			print_error ptr
+		end
+		failed_parsing.add src
```

Comment:
I agree that you should move up this in nitcc_runtime: A service in `Position` that does the printing of the line with the carret and a service in NError that does the whole printing.